core/src/main/scala/cromwell/core/callcaching/CallCachingMode.scala

```diff
@@ -13,6 +13,8 @@ sealed trait CallCachingMode {
 }
 
 case object CallCachingOff extends CallCachingMode {
+  override val readFromCache = false
+  override val writeToCache = false
```

Comment:
I reckon it should be fine... it just doesn't change the default behaviour of `CallCachingMode` at all. Maybe it doesn't hurt to be explicit!